roles/schema_registry/tasks/main.yml

```diff
@@ -173,6 +173,12 @@
     - schema_registry_secrets_protection_enabled|bool
     - rbac_enabled|bool
 
+- name: Set Permissions on /var/log/confluent
```

Comment:
Do we have any confirmation form Schema Registry team?

Reply:
Basically, in a custom (Linux) user and groups scenario, we do need this behaviour which makes this parent logs directory accessible to the user of the service.
In default cases, its all good as permissions default user `cp-ansible/confluent` works for all services. It digresses only when different services need to be run with different user/groups.
Colocated services might need more careful considerations here.

Reply:
I'm not sure why the service is trying to write in this location. Given we do provide the log path and we do assign appropriate rights for custom users. This is what I wanted to get clarity from the component team, under what conditions, the services try to write on the above location.

Reply:
This is parent directory of log location, by the way.
So if we leave log location as default, it tries to write into a directory under here. 
`/var/log/confluent/schema-registry` in this case.
Now we give permission to above mentioned directory, but we don't give permission on parent directory i.e. `/var/log/confluent`